backend/routes/movieServices.js

```diff
@@ -119,9 +120,21 @@ async function getSuggestedMovies(userId) {
   //! Wait on user schema to implement
   try {
     if (userId) {
-      const movieId = 0;
+      //Create ObjectId of user's Id, so findOne can successfully return a matching review
```

Comment:
We should finish the test cases for this function. They should be at very bottom of the movieService.test.js file.